app/lib/models/EditGroupInfoModel.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:effektio_flutter_sdk/effektio_flutter_sdk_ffi.dart';
+
+class EditGroupInfoModel {
```

Comment:
Are these models needed for ```beamer``` functionality?

Reply:
Yes, to pass data into beamer we have to pass it in data key which takes "data as MyObject" where MyObject is a class. So these models are used there, Regardless wherever there is only one parameter to pass, I passed it directly without model class